server/auth/jwt.go

```diff
@@ -21,7 +21,7 @@ import (
 	"errors"
 	"time"
 
-	jwt "github.com/form3tech-oss/jwt-go"
+	jwt "github.com/golang-jwt/jwt"
```

Comment:
After bumpping to 4.4.2, we need to import `"github.com/golang-jwt/jwt/v4"` instead. 

FYI. https://github.com/golang-jwt/jwt/blob/bfea432b1a9da383509086a2141c06dc103e82f9/cmd/jwt/main.go#L20